### ISSUE TITLE
Feature request - allow throwing rocks.

### ISSUE THREAD

**Issue by K0-RR** [TeamMidnightDust/ThisRocks#20]
Like snowballs.

**Comment by Motschen** [comment]
I don't think this would fit my vision for the mod, but I'll leave it up for debate.

**Comment by NealSavage** [comment]
I like the skipping rock function in the Campanian mod. Perhaps you could make the rocks in this mod work as skipping rocks with that mod, or you can just make them work as skipping rocks vanilla (as in, only with ThisRocks without Campanian.)
